pattern_import_export/models/ir_exports_line.py

```diff
@@ -209,3 +230,68 @@ def _get_header(self, use_description=False):
                                 )
                             )
         return headers
+
+    def _get_tab_headers(self):
+        self.ensure_one()
+        return [self.last_field_id.name]
+
+    def _format_tab_records(self, permitted_records):
+        return [
+            [getattr(record, self.last_field_id.name)] for record in permitted_records
+        ]
+
+    def _get_tab_data(self):
+        """
+        :return: iterable of 4-tuples of format:
+        (name, headers, data, origin_col)
+        one tuple for each tab
+        name: sheet name
+        headers: list of strings, each element mapping to one header cell
+        data: list of lists, each element mapping to one row/cells
+        origin_col: position of the column on the main sheet
+        """
+        result = []
+        for itr, rec in enumerate(self, start=1):
+            if not (
+                rec.related_model_relation_type in ("many2many", "many2one")
```

Comment:
It will be better to only test the add_select_tab and have a correct value for this field.
When we change the last field we should untick the add_select_tab in some case (we should also have this behaviour for the onchange)